ui/hooks/useTokenFiatAmount.js

```diff
@@ -67,8 +67,8 @@ export function useTokenFiatAmount(
   );
 
   const tokenConversionRate = chainId
-    ? currencyRates[networkConfigurationsByChainId[chainId].nativeCurrency]
-        .conversionRate
+    ? currencyRates?.[networkConfigurationsByChainId[chainId]?.nativeCurrency]
+        ?.conversionRate
     : conversionRate;
```

Comment:
Were you able to repro the linked issue locally? 
From the error linked in the issue i think this optional chaining makes sense

Reply:
I wasn't able to reproduce exactly, but I've definitely seen similar bugs from time to time related to currencyRates not yet resolving.